runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -198,10 +219,37 @@ pub enum Cost {
     /// DEPRECATED: Was charged in `storage_iter_next`
     StorageIterNextValueByte,
     TouchingTrieNode,
+    /// Estimates `promise_and_base` which is charged for every call to
+    /// `promise_and`. This should cover the base cost for creating receipt
+    /// dependencies.
+    ///
+    /// Estimation: Currently not estimated
     PromiseAndBase,
+    /// Estimates `promise_and_per_promise` which is charged for every promise in
+    /// calls to `promise_and`. This should cover the additional cost for each
+    /// extra receipt in the dependency.
+    ///
+    /// Estimation: Currently not estimated
     PromiseAndPerPromise,
+    /// Estimates `promise_return` which is charged when calling
+    /// `promise_return`. This should cover the cost of the dependency between a
+    /// promise and the current function call return value.
+    ///
+    /// Estimation: Currently not estimated
     PromiseReturn,
+    /// Estimates `validator_stake_base` which is charged for each call to
+    /// `validator_stake`, covering the cost for looking up if an account is a
+    /// validator and if so, how much it has staked. This information is
+    /// available from the local EpochManager.
+    ///
+    /// Estimation: Currently not estimated
     ValidatorStakeBase,
+    /// Estimates `validator_total_stake_base` which is charged for each call to
+    /// `validator_total_stake`, covering the cost for looking up the total
+    /// staked tokens for the current epoch. This information is
+    /// available from the local EpochManager.
+    ///
+    /// Estimation: Currently not estimated
```

Comment:
These seem like they ideally want to be a separate commit, but since merging will squash the commits anyway, I guess its fine to leave as is.